pydantic/_hypothesis_plugin.py

```diff
@@ -0,0 +1,169 @@
+"""Register Hypothesis strategies for Pydantic custom types.
```

Comment:
```suggestion
"""
Register Hypothesis strategies for Pydantic custom types.
```

(sorry, my pet dislike of docstrings with no leading blank line.)